README.md

```diff
@@ -58,9 +58,64 @@ Note for Ubuntu: Please note that the uncomplicated firewall (ufw) is known to c
 
 ## Deployment
 
+### Site name
+
+You will need to choose a short name for your site. This is not a URL, just a simple identifying string. For the examples below, we will use "your-site-name", but you should obviously choose something that is meaningful to you and which is unique.
+
+Site names should adhere to the following conventions:
+
+- They should be lower-case.
+- They should generally be named after the city where your site is based, e.g. ```karlsruhe```.
+- If you have a multi-part name, please use a hypen ("-") as separator, e.g. ```le-havre```.
+- If your site is for testing purposes, rather than production, please append "-test", e.g. ```zaragoza-test```.
+- If you are a developer and you are making changes to the Bridgehead, please use your name and append "-dev", e.g. ```joe-doe-dev```.
+
+### Projects
+
+The following "projects" are known to the Bridgehead installation:
+
+- bbmri
+- ccp
+
+Use "bbmri" if you are in the BBMRI-ERIC European biobank network or the GBA (German Biobank Alliance) network.
+
+Use "ccp" if you are in the DKTK network, the C4 network or the nNGM network.
+
+### GitLab repository
+
+In order to be able to install, you will need to have your own repository in GitLab for your site's configuration settings. This allows automated updates of the Bridgehead software.
+
+To request a new repository, please send an email to one of the following:
+
+- For the bbmri project: bridgehead@helpdesk.bbmri-eric.eu.
+- For the ccp project: support-ccp@dkfz-heidelberg.de
+
+Mention which project you belong to, i.e. "bbmri" or "ccp", plus your chosen site name.
+
+We will set the repository up for you. We will then send you:
+
+- A Repository Short Name (RSN). Beware: this is distinct from your site name.
+- The repository's URL.
+- A token to access the repository.
```

Comment:
```suggestion
- Your site name (e.g. ```karlsruhe```).
- An access token to access the repository.
```

I believe these two infos should be all that is required to complete the installation.